docs/developer-docs/gas-cost.md

```diff
@@ -4,15 +4,107 @@
+
+- 1_000 ingress messages received per month with 1_000 bytes per message: 3_200_000_000 cycles. 
+
+**Approximate monthly cost:** 13_812_225_000_000 cycles; 13.812225152 XDR. At the exchange rate of 1.308860, this would cost $18.08 USD.
+
+### Simple static website with a frontend and backend canister 
+
+For a simple static website that uses two canisters (one for the website's backend functionality, one for the frontend assets), that is not called by other canisters or performs HTTPS outcalls, the following approximate cycles costs would be incurred:
+
+- Canister creation for two canisters (one-time fee): 200_000_000_000 cycles.
+
+- Storing 8GB of data (across both canisters): 1_016_000 cycles per second, 2_670_340_600_000 cycles per month.
+
+- Using 2% compute resources: 20_000_000 cycles per second; 52_565_760_000_000 cycles per month.
+
+**Approximate monthly cost:** 55_236_101_000_000 cycles; 55.2361006 XDR. At the exchange rate of 1.308860, this would cost $72.30 USD.
+
+### Social media dapp with two canisters
+
+For a project that creates a social media dapp using two canisters with around 200 active user accounts that interact with the dapp monthly, the following approximate cycles costs would be incurred:
+
+- Canister creation for two canisters (one-time fee): 200_000_000_000 cycles.
+
+- Storing 25GB of data: 3_175_000 cycles per second; 8_344_814_400_000 cycles per month.
+
+- Using 5% compute resources: 50_000_000 cycles per second; 131_414_400_000_000 cycles per month.
+
+- 500_000 ingress messages with 1_000 bytes per message (2_500 per active user, per month): 600_002_000_000 cycles.
+
+**Approximate monthly cost:** 140_359_220_000_000 cycles; 140.3592164 XDR. At the exchange rate of 1.308860, this would cost $107.24 USD.
+
+### Decentralized service using threshold ECDSA and HTTPS outcalls
+
+For a project that creates a decentralized service that makes several thousand HTTPS outcalls to services outside ICP, utilizes the threshold ECDSA feature on ICP, and makes several thousand inter-canister calls, the following approximate cycles costs would be incurred:
+
+- Canister creation for ten canisters (one-time fee): 1_000_000_000_000 cycles.
+
+- Storing 150GB of data: 19_050_000 cycles per second; 50_068_886_000_000 cycles per month. 
+
+- Using 10% compute resources: 100_000_000 cycles per second; 262_828_800_000_000 cycles per month.
+
+- 100_000 ingress messages with 1_000 bytes per message: 320_000_000_000 cycles.
+
+- [5_000 threshold ECDSA signature keys](/docs/current/developer-docs/integrations/t-ecdsa/t-ecdsa-how-it-works#fees-for-the-t-ecdsa-production-key) per month: 130_769_230_000_000 cycles. 
+
+- 45_000 HTTPS outcalls requests with 800 bytes per message: 2_211_534_000_000 cycles.
+
+- 45_000 HTTPS outcalls responses with 800 bytes per message: 468_000_000 cycles.
+
+**Approximate monthly cost:** 445_878_920_000_000 cycles; 445.87892 XDR. At the exchange rate of 1.308860, this would cost $583.59 USD.
```

Comment:
we may want to provide some perspective here (similar to putting a banana in a picture for scale). ~$600 a month may be prohibitively expensive at small scale, but may be a very attractive figure if scaled similarly against a web2 stack whose cost would be in the thousands.

Reply:
Revised to include perspective regarding how this example showcases an enterprise architecture and used Orally as a linked example of an enterprise that uses a similar workflow (HTTPS outcalls, tECDSA, etc).